internal/pkg/action/dispatcher.go

```diff
@@ -122,6 +124,14 @@ func (d *Dispatcher) process(ctx context.Context, hits []es.HitT) {
 	}
 
 	for agentID, actions := range agentActions {
+		if d.throttle != 0 {
+			select {
+			case <-ctx.Done():
+				return
+			case <-time.After(d.throttle):
```

Comment:
The reference documentation above doesn't make it clear that the throttle delay is cumulative for each agent dispatching an action.